src/server/database/Database/MySQLConnection.h

```diff
@@ -35,6 +35,17 @@ enum ConnectionFlags
     CONNECTION_BOTH = CONNECTION_ASYNC | CONNECTION_SYNCH
 };
 
+struct PreparedStatementInfo
+{
+    PreparedStatementInfo(std::string const& queryString, ConnectionFlags connFlags) :
+        Query(queryString), Flags(connFlags) { }
+
+    std::string const Query;
+    ConnectionFlags const Flags; // sync/async
```

Comment:
On a second thought, stored connection flags for the Prepared statement are not used at all.

Query string could be moved to MySQLPreparedStatement class, and remove this container entirely, so there is only one container, and it's a vector so it's fast